ontopy/utils.py

```diff
@@ -21,7 +21,7 @@
 
 
 if TYPE_CHECKING:
-    from packaging.version import Version, LegacyVersion
+    from packaging.version import Version
```

Comment:
Move this up and out of the `TYPE_CHECKING` block as `Version` is used explicitly in line 454 and not just as a type hint.